backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5637900_sys_gh11302_process_remove_old_hu.sql

```diff
@@ -0,0 +1,131 @@
+;
+
+-- 2022-05-05T07:21:58.970Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Process_Para_Trl (AD_Language,AD_Process_Para_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Process_Para_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Process_Para t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y') AND t.AD_Process_Para_ID=542257 AND NOT EXISTS (SELECT 1 FROM AD_Process_Para_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Process_Para_ID=t.AD_Process_Para_ID)
+;
+
+-- 2022-05-05T07:38:13.545Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Scheduler (AD_Client_ID,AD_Org_ID,AD_Process_ID,AD_Role_ID,AD_Scheduler_ID,Created,CreatedBy,EntityType,Frequency,FrequencyType,IsActive,IsIgnoreProcessingTime,KeepLogDays,ManageScheduler,Name,Processing,SchedulerProcessType,ScheduleType,Status,Supervisor_ID,Updated,UpdatedBy) VALUES (0,0,585049,0,550087,TO_TIMESTAMP('2022-05-05 09:38:13','YYYY-MM-DD HH24:MI:SS'),100,'D',1,'D','Y','N',7,'N','remove_hu_entries_between_dates','N','P','F','NEW',100,TO_TIMESTAMP('2022-05-05 09:38:13','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2022-05-05T07:38:33.308Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Scheduler_Para (AD_Client_ID,AD_Org_ID,AD_Process_Para_ID,AD_Scheduler_ID,AD_Scheduler_Para_ID,Created,CreatedBy,IsActive,ParameterDefault,Updated,UpdatedBy) VALUES (0,0,542257,550087,540038,TO_TIMESTAMP('2022-05-05 09:38:33','YYYY-MM-DD HH24:MI:SS'),100,'Y','100',TO_TIMESTAMP('2022-05-05 09:38:33','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2022-05-05T07:38:56.861Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Scheduler_Para (AD_Client_ID,AD_Org_ID,AD_Process_Para_ID,AD_Scheduler_ID,AD_Scheduler_Para_ID,Created,CreatedBy,IsActive,ParameterDefault,Updated,UpdatedBy) VALUES (0,0,542256,550087,540039,TO_TIMESTAMP('2022-05-05 09:38:56','YYYY-MM-DD HH24:MI:SS'),100,'Y','24',TO_TIMESTAMP('2022-05-05 09:38:56','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2022-05-05T08:07:28.389Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+UPDATE AD_Process SET Description='Removes all handling unit entries that have an inactive status and are older than the specified months. The placeholder is used in other tables linking to the removed handling unit.',Updated=TO_TIMESTAMP('2022-05-05 10:07:28','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Process_ID=585049
```

Comment:
nice, i think you can reuse this also for the comment i requested further up

LE: please write something like "deletes from the DB" instead of "removes"..needs to be as clear as possible that stuff is deleted

Reply:
...please also add a DE translation for the name and description.